cmd/fsck.go

```diff
@@ -64,6 +64,11 @@ $ juicefs fsck redis://localhost --path /d1/d2 --recursive`,
 				Aliases: []string{"r"},
 				Usage:   "recursively check or repair",
 			},
+			&cli.BoolFlag{
+				Name:    "stat-all",
+				Aliases: []string{"s"},
```

Comment:
no short alias if it's not used frequently

Reply:
Removed